### ISSUE TITLE
PORTUS_DB_ADAPTER is not overriding default mysql2

### ISSUE THREAD

**Issue by mnick-nm** [SUSE/Portus#1687]
I rebuild from opensuse/portus:2.3 every few days to get any updates.  In the last 24 hours I still have PORTUS_DB_ADAPTER being set to postgresql in my kube.yml file.  I can verify that the env var exists as PORTUS_DB_ADAPTER=postgresql in the container after it starts.

But for some reason its not overriding the default anymore and my start up logs show:
```Waiting for mariadb to be ready in 5 seconds
Waiting for mariadb to be ready in 5 seconds
Waiting for mariadb to be ready in 5 seconds
Waiting for mariadb to be ready in 5 seconds
```

I thought it was some other setting that I altered, but I think there is a bug in portus:2.3.

The database.yml appears to confirm this:
```cat database.yml 
default: &default
  adapter: mysql2
  encoding: utf8
<% if ENV['COMPOSE'] %>
  host: <%= ENV['PORTUS_DB_HOST'] %>
  username: root
  password: portus
<% end %>
```

**Comment by mnick-nm** [comment]
the way the code looks, I thought adapter was determined by the env var

**Comment by mnick-nm** [comment]
I reverted to a previous tag (I build from portus2.3 and copy in my configs/vars) and it works with adapter tag, but the background container complains that `Registry integration has been disabled. This is highly discouraged!
[Initialization] Running: 
/srv/Portus/bin/background.rb:34:in `map': undefined method `sleep_value' for nil:NilClass (NoMethodError)
	from /srv/Portus/bin/background.rb:34:in `<top (required)>'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands/runner.rb:60:in `load'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands/runner.rb:60:in `<top (required)>'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands/commands_tasks.rb:123:in `require'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands/commands_tasks.rb:90:in `runner'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /srv/Portus/vendor/bundle/ruby/2.5.0/gems/railties-4.2.10/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:12:in `require'
	from bin/rails:12:in `<main>'
bundler: failed to load command: packaging/suse/portusctl/bin/portusctl (packaging/suse/portusctl/bin/portusctl)
RuntimeError: Something went wrong while invoking: /srv/Portus/vendor/bundle/ruby/2.5.0/bin/bundler.ruby2.5 exec rails r /srv/Portus/bin/background.rb
  /srv/Portus/packaging/suse/portusctl/lib/runner.rb:12:in `exec'
  /srv/Portus/packaging/suse/portusctl/lib/runner.rb:34:in `block in bundler_exec'
  /srv/Portus/packaging/suse/portusctl/lib/runner.rb:30:in `chdir'
  /srv/Portus/packaging/suse/portusctl/lib/runner.rb:30:in `bundler_exec'
  /srv/Portus/packaging/suse/portusctl/lib/cli.rb:272:in `exec'
  /srv/Portus/vendor/bundle/ruby/2.5.0/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
  /srv/Portus/vendor/bundle/ruby/2.5.0/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
  /srv/Portus/vendor/bundle/ruby/2.5.0/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
  /srv/Portus/vendor/bundle/ruby/2.5.0/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
  packaging/suse/portusctl/bin/portusctl:6:in `<top (required)>'
`

**Comment by mnick-nm** [comment]
I was making mistakes and this issue was my problem > @ironhouzi you might be hitting a regression we had yesterday due to #1666, which we fixed later with #1668. So, between #1666 and #1668
